scripts/spack/packages/amgx/package.py

```diff
@@ -21,9 +21,13 @@ class Amgx(CMakePackage, CudaPackage):
     git      = "https://github.com/NVIDIA/AMGX.git"
 
     maintainers = ['js947']
+
     # SERAC EDIT BEGIN
-    version('2.1.x', commit='c5405f3f18fbe278292ed0226d970992857e266f')
+    # white238: CUSPARSE_CSRMV_ALG2 undefined error in 2.3.0
+    version("2.3.0.1", commit="d2344958f43c103893c4400fe8ad42d02ac773f5", submodules=True)
     # SERAC EDIT END
+    version("2.3.0", sha256="419b3cd5bd3eb3469cbef79d64a8d19d5db88dd5cce809e49cac6fc4fc2edff1")
```

Comment:
I pushed these two new version to spack but that didn't fix the issue and it was resolved in their develop branch. so that is the .1 version.